File: DisplaFrontend/src/app/model/lente.ts
import { Usuario } from './usuario';
import { PrecioLente } from './precioLente';
import { RecargoLente } from './recargoLente';

export interface Lente {
    Id: number,
    Nombre: string,
    DescripcionFactura: string,
    Combinacion: string,
    GraduacionesCilindricas: string,
    MediosPares: boolean,
    IdUsuario: number,
    IdUsuarioNavigation: Usuario,
    FechaCreacion: Date,
    IngresosBrutos: number,
    Borrado: boolean
    PrecioLente: PrecioLente[],
    RecargoLente: RecargoLente[],
    Iva: number,
    Fraccionado: boolean
}

File: DisplaBackend/DisplaBackend/Models/Lente.cs
using System;
using System.Collections.Generic;

namespace DisplaBackend.Models
{
    public partial class Lente
    {
        public Lente()
        {
            ComprobanteItemLente = new HashSet<ComprobanteItemLente>();
            PrecioLente = new HashSet<PrecioLente>();
            RecargoLente = new HashSet<RecargoLente>();
            VentaVirtual = new HashSet<VentaVirtual>();
        }

        public int Id { get; set; }
        public string Nombre { get; set; }
        public string DescripcionFactura { get; set; }
        public string Combinacion { get; set; }
        public string GraduacionesCilindricas { get; set; }
        public bool? MediosPares { get; set; }
        public int? IdUsuario { get; set; }
        public DateTime? FechaCreacion { get; set; }
        public double? IngresosBrutos { get; set; }
        public bool Borrado { get; set; }

        public virtual ICollection<ComprobanteItemLente> ComprobanteItemLente { get; set; }
        public virtual ICollection<PrecioLente> PrecioLente { get; set; }
        public virtual ICollection<RecargoLente> RecargoLente { get; set; }
        public virtual ICollection<VentaVirtual> VentaVirtual { get; set; }
    }
}
